Documentation/ColumnsConfig/Type/Language/Introduction.rst

```diff
@@ -0,0 +1,32 @@
+.. include:: /Includes.rst.txt
+.. _columns-languge-introduction:
+
+============
+Introduction
+============
+
+A new TCA field type called `language` has been added to TYPO3 Core. It's main
```

Comment:
Please adjust the text from the changelog to avoid references to "new" - in case of this introduction I would split it in two parts: one that is a real introduction (what is this field, what does it do) and one that describes the history for people who are working with TYPO3 since a while longer. This allows users that are new or are "just here for the info" to skim the chapter and find what they need quickly.